python/social_bot/envs/play_ground.py

```diff
@@ -125,7 +125,7 @@ def __init__(self,
         self._action_cost = action_cost
         self._with_language = with_language
         self._seq_length = vocab_sequence_length
-        self._with_agent_language = with_agent_language
+        self._with_agent_language = with_language and with_agent_language
```

Comment:
it seems we should add this logic to prevent the case with_language is false and with_agent_language is true.

Perhaps we can change them to something like with_language_in_observation and  with_language_in_action

Reply:
Thanks for the fix Jiangtao, LG.  If we have more than 2 sources of language then at that point we can do _in_obs, _in_action etc.?